### ISSUE TITLE
Where to buy the PCB for this project?

### ISSUE THREAD

**Issue by bwims** [opaJG/Z99#3]
Hello,

surprisingly, it seems that Elektor do not sell the PCB for this project.  It's been many years since I last ordered one (for the minidisc scmp decoder, back in the 90s) so I guess things might have changed.

There was no mention in your book as to how to get the PCB.  Is it an exercise for the reader?  The main reason I'd like to build this is to enable me to make PCBs...

Thanks!

B.

**Comment by bwims** [comment]
Hmm.  Sell an expensive book, supply NO details about procuring the essential PCB required to develop the project, unlike any other Elektor project from the 70s onwards, create a github "support" account, and do absolutely F*ck All with it.

**Comment by Mimakur** [comment]
> Hello,
> 
> surprisingly, it seems that Elektor do not sell the PCB for this project. It's been many years since I last ordered one (for the minidisc scmp decoder, back in the 90s) so I guess things might have changed.
> 
> There was no mention in your book as to how to get the PCB. Is it an exercise for the reader? The main reason I'd like to build this is to enable me to make PCBs...
> 
> Thanks!
> 
> B.

Hello. I know this was asked many months ago, but if it still helps you or maybe someone else. This is one way to order pcb for this project.

Open the installation location, there is folder named PCB_LayoutAndSchematicDesign --> open it and now you should see two folders. Here DesignsparkPCB folder contains what you are looking for. It has Readme file and also a zip file called X2Y4Z.

Basically you go to for example jlcpcb or pcbway website, upload/send that X2Y4Z file where they ask "Gerber file". After that it should be ready to order, maybe double check all the parameters and everythings it offers since i haven't actually done this (yet) for this project.

You also need to find, select and order all the required components for this project. There are many websites for components (lcsc, mouser, farnell etc etc.. ). I'd recommend that you download KiCad software and learn the schematic for this project.

**Comment by bwims** [comment]
Thanks for the advice.  I guess it all boils down to: the author thinks if you can't order the pcb, you are not up to making the project.... to which I reply, I don't want to go through all that nausea when I don't even know if the project works or is any good!
I haven't found any endorsements for this project anywhere...

**Comment by Mimakur** [comment]
Here's video i found made by author: https://www.youtube.com/watch?v=yEihIBTxz5Q

Based on that video it seems to me that it is functional.

I think it is an interesting project and that's why i'm re-designing this pcb for my own liking. I am changing all of the components to smd versions, but i'm also going to order pcb made of that gerber file x2y4z just in case which needs through-hole components.
